README.md

```diff
@@ -9,8 +9,31 @@ It's easy! Just create a directory in the top level for your raffler.
 
 Each raffler should accept a filename as the first CLI argument. The file will contain a line-delimited list of names, from which the raffler should echo a single random name from. See the example_names file for an example of the format.
 
-You can write it in any language (provided you can prove to our hosts that it's not malware). If you've got dependencies, please include a simple install.sh for the debian dependencies.
+You can write it in any language. The weirder the better If you like insane over engineering, do so! If you like to do it the way you've been telling your junior colleagues no to, go ahead!
 
 ## Getting Merged Quickly
-- Please throw in an install.sh if you're using something besides PHP or Python.
+- Please supply a Dockerfile so we do not have to install al these weird stuff on our systems directly :-P 
 - Wacky GIFs aren't required in your PR but are encouraged (see YoloSR-2).
+
+## Dockerfile example:
+```
+FROM ubuntu:15.10 # or a more suitable base container
+MAINTAINER yourname@domain.org
+ENV LANG C.UTF-8
+
+# Install your dependencies
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y deps-you-require
+
+# Create working dir
+RUN mkdir -p /var/app
+COPY . /var/app
+WORKDIR /var/app
+
+# Compile raffler (if necessary)
+RUN /var/app/compile.sh # everything needed to compile your raffler
+
+# Run raffler
+CMD ["/var/app/run.sh"] #everything needed to run your raffler
```

Comment:
That's what I did, since mine worked through a shebang for the `.as` itself, I chmodded `run.sh` and `raffler.as`.

But you could just `CMD ["bash", "/var/app/run.sh"]`, which implies `run.sh` is a bash script, in which the implementer itself is responsible for additional chmods if needed (Like I needed, but oftenly not)